src/components/SingleCategory.jsx

```diff
@@ -0,0 +1,34 @@
+import { Link, useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchReviewsByCategories } from "../utils/API";
+
+export const SingleCategory = () => {
+  const { slug } = useParams();
+  const [SingleCategory, setSingleCategory] = useState([]);
+  useEffect(() => {
+    fetchReviewsByCategories(slug).then((reviewCategory) => {
+        console.log(reviewCategory);
+      setSingleCategory(reviewCategory);
+    });
+  }, [slug]);
+
+  return (
+    <ul className="review-card">
+      {SingleCategory.map((chosenReview) => {
+        return <div>
+            <li className="itemCard" key={chosenReview.review_id}>
```

Comment:
symantically the first child of a `<ul>` or `<ol>` should be a `<li>` (not a `<div>`)